scripts/pytest-image-runner.sh

```diff
@@ -34,12 +36,11 @@ build_pytest_image() {
     local __image_tag=$1
 
     local ack_role_arn=$(get_assumed_role_arn)
```

Comment:
have I mentioned it is confusing to have an AWS_ROLE_ARN and an ACK_ROLE_ARN? :)

Reply:
I totally agree. `AWS_ROLE_ARN` is supplied to the test container by IRSA. `ACK_ROLE_ARN` is an unfortunately named variable that I have only kept for the sake of backward compatibility. I'd be happy to change it if we're willing to break that.

Reply:
Give me 10 minutes

Reply:
Updated and also created https://github.com/aws-controllers-k8s/community/pull/1399